test/e2e/atlas/compliancepolicy_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build e2e || (atlas && backup && compliancepolicy)
+
+package atlas_test
+
+import (
+	"strings"
+	"testing"
+)
+
+const (
+	authorizedEmail = "firstname.lastname@example.com"
+)
+
+func TestCompliancePolicy(t *testing.T) {
+	g := newAtlasE2ETestGenerator(t)
+	g.generateProject("compliancePolicy")
+
+	testCompliancePolicySetup(t, g)
+	testDescribe(t, g)
+	testPoliciesDescribe(t, g)
+	testCopyProtection(t, g)
```

Comment:
what's the relationship between these tests, do they need to run in this order?

Reply:
Setup has to be first, others are independent.

Reply:
> others are independent.

why do we need to run them in the same test?

Reply:
I mean't independent in terms of order.

Reply:
I'm going with `r` and `a`, since that's the pattern used in the other E2E tests.